WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrder.swift

```diff
@@ -158,6 +155,16 @@ private struct ProductsSection: View {
             Divider()
         }
     }
+
+    @ViewBuilder private func createProductInOrderView(for item: NewOrderViewModel.NewOrderItem) -> some View {
+        if let productRowViewModel = viewModel.createProductRowViewModel(for: item, canChangeQuantity: false) {
+            let productInOrderViewModel = ProductInOrderViewModel(productRowViewModel: productRowViewModel) {
+                viewModel.removeItemFromOrder(item)
+            }
+            ProductInOrder(viewModel: productInOrderViewModel)
+        }
+        EmptyView()
```

Comment:
Q: Any specific reason to have EmptyView here?

Reply:
Maybe it can be placed in "else" clause then?
Also, I've tried commenting it out, and it works.